packages/angular_devkit/build_angular/package.json

```diff
@@ -33,13 +33,13 @@
     "karma-source-map-support": "1.4.0",
     "less": "3.10.3",
     "less-loader": "5.0.0",
-    "license-webpack-plugin": "2.1.3",
+    "license-checker-webpack-plugin": "^0.1.4",
```

Comment:
Please pin down

Reply:
Done. I left the `@types/...` definition alone since that's just a dev dependency. Should we also lock down that version as well, or is this sufficient?

Reply:
That is sufficient. Thanks